backend/Dockerfile.backend

```
@@ -5,6 +5,9 @@ ENV PYTHONPATH=/backend
 
 WORKDIR /backend
 
+# FFmpegのインストール
+RUN apt-get update && apt-get install -y ffmpeg
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)